app/src/main/res/layout/question_player_activity.xml

```diff
@@ -20,7 +21,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_gravity="end"
-        android:background="@color/oppia_primary"
+        android:background="@color/color_def_oppia_green"
```

Comment:
This should be a color from component_colors.xml

Reply:
Done